platform.txt

```diff
@@ -101,10 +101,6 @@ recipe.output.save_file_hex={build.project_name}.save.hex
 recipe.size.pattern="{compiler.path}{compiler.size.cmd}" -A "{build.path}/{build.project_name}.elf"
 recipe.size.regex=\.text\s+([0-9]+).*
 
-## Export Compiled Binary
-recipe.output.tmp_file={build.project_name}.hex
-recipe.output.save_file={build.project_name}.{build.variant}.hex
```

Comment:
I think we need these too? Added in https://github.com/sandeepmistry/arduino-nRF5/commit/54028f795b41e4d1e6fb708ddfca429bdd45ea2d